packages/bulkapi/src/BulkApiContext.ts

```diff
@@ -32,6 +32,8 @@ export function registerBulkApi(app: Express, dbConnection: pgPromise.IDatabase<
     const context = new BulkApiContext(dbConnection, pgp)
     
     // Add routes to application
+    app.post("/bulk/createPartitions", runWithTry(context.bulkApi.createPartitions))
```

Comment:
Why `create` and `delete` partition**s** (plural)?

Reply:
Because that is how they are called in the current specification :-). See [specification](https://github.com/LionWeb-io/specification/blob/niko/bulk-api-spec/bulk/repo-access-api.adoc)